doc/man7/ossl-guide-quic-server-non-block.pod

```diff
@@ -0,0 +1,369 @@
+=pod
+
+=begin comment
+
+NB: Changes to the source code samples in this file should also be reflected in
+demos/guide/quic-server-non-block.c
+
+=end comment
+
+=head1 NAME
+
+ossl-guide-quic-server-non-block
+- OpenSSL Guide: Writing a simple nonblocking QUIC server
+
+=head1 SIMPLE NONBLOCKING QUIC SERVER EXAMPLE
+
+This page presents various source code samples demonstrating how to write a
+simple, non-concurrent, QUIC "echo" server application which accepts one client
+connection at a time, echoing input from the client back to the same client.
+Once the current client disconnects, the next client connection is accepted.
+
+The server only accepts HTTP/1.0 requests, which is non-standard and will not
```

Comment:
I don't think this sample is doing any http except matching the string in the alpn?

Reply:
Wouldn't echoing the client data be considered HTTP?

Reply:
No, even ignoring the more complex parts of http, the client to server and the server to client parts differ in essential syntax.
The clients sends something beginning with `GET / HTTP/1.1` and the server sends something beginning with `HTTP/1.1 200 OK`.

So an echo application is more something to use with openssl s_client or a hypothetical version of socat with quic support.

Reply:
So we can remove the reference to HTTP/1.0 and just change it to this?

> The server only accepts `http/1.0` and `hq-interop` ALPN's, which is non-standard...

Reply:
maybe something to that effect.
You might want to add an explicit "and doesn't actually implement http but only does a simple echo" part.
Maybe it would be useful to add a line how to test this server using openssl s_client or maybe you have another nice any easy way to play with it.